Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -721,7 +721,11 @@ public final class SwiftModuleBuildDescription {
 
     /// When `scanInvocation` argument is set to `true`, omit the side-effect producing arguments
     /// such as emitting a module or supplementary outputs.
-    public func emitCommandLine(scanInvocation: Bool = false) throws -> [String] {
+    /// If `writeOutputFileMap` is `false`, we assume that an output file map for this command line already exists at
+    /// the expected location on disk. This is intended for SourceKit-LSP to get build settings for a file without
+    /// writing out an output file map as a side effect. We expect that preparation of the module has already created
+    /// the output file map.
```

Comment:
Can this be rewritten in a more usual doc comments format for clarity?

```suggestion
    /// <#Description#>
    /// - Parameters:
    ///   - scanInvocation: <#scanInvocation description#>
    ///   - writeOutputFileMap: <#writeOutputFileMap description#>
    /// - Returns: <#description#>
```